config/routes.rb

```diff
@@ -6,4 +6,8 @@
     get '/orders/to_export', to: 'orders#to_export', as: :orders_to_export
     put '/orders/:id/export', to: 'orders#export', as: :order_export
   end
+
+  namespace :admin do
+    get '/xlsx-export/orders', to: 'orders#export_xlsx', as: :order_export_xlsx
```

Comment:
Ah ok. If it is only for Jenkins does the route need to live in the admin namespace at all?